src/index.js

```diff
@@ -0,0 +1,23 @@
+import './style.css';
+import '@fortawesome/fontawesome-free/css/all.css';
+import Tasklist from '../modules/tasklist.js';
+import addItem from '../modules/addItem.js';
+import displayItem from '../modules/displayItem.js';
+
+class App {
+  constructor() {
+    this.myTasklist = new Tasklist();
+    this.myTasklist.addBtn.addEventListener('click', this.myAddItem.bind(this));
+  }
+
+  myDisplayItem() {
+    displayItem(this.myTasklist.listOfTasks);
+  }
+
+  myAddItem() {
+    addItem(this.myTasklist.listOfTasks);
+  }
+}
+
+const app = new App();
+app.myDisplayItem();
```

Comment:
- [ ] Impressive work. Could you consider following the project requirements? Your app is working fine but it doesn't adhere to the project requirement of this milestone. In this milestone, you're required to do a  setup of your project and create an array of objects with keys including:
- description `[string]`.
- completed `[boolean]`.
- index: `[number]`.

and write a function to `iterate` over the tasks array and populate them to the UI based on their `index`. 
- [ ] Kindly note that it's okay to have the add and remove functionalities as you're going to be doing all that in upcoming milestones but for this specific milestone, please make sure to implement everything as specified in the project requirement. See below for more details on the project requirements:
<img width="1003" alt="Screenshot 2023-06-08 at 11 06 49" src="https://github.com/Chandan-devs-tech/To-Do-list/assets/107192076/f5842deb-4be8-4713-8a5c-2556ba2b3eb6">